proposals/XX-bw-xxx.txt

```diff
@@ -0,0 +1,202 @@
+          Tor Bandwidth Measurements Document Format
+                            juga
+                            teor
+
+1. Scope and preliminaries
+
+  This document describes the format of Tor's bandwidth measurements
+  document, version 1.0.0 and later.
+
+  Since Tor version 0.2.4.12-alpha the directory
+  authorities use the bandwidth measurements document called
+  "V3BandwidthsFile" and produced by Torflow [1]
+  (format described in README.spec.txt [2]).
+
+    The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+    NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+    "OPTIONAL" in this document are to be interpreted as described in
+    RFC 2119.
+
+1.2. Acknowledgements
+
+  The original bandwidth measurement scanner (Torflow) and format was
+  created by mike. Teor suggested to write this specification while
+  contributing on pastly's new bandwidth scanner implementation.
+
+  This specification was revised after feedback from:
+
+    XXX
+
+1.3 Outline
+
+  The bandwidth measurements mentioned in sections 3.4.1 and 3.4.2
+  of "Tor directory protocol" (dir-spec.txt) [3] are obtained
+  by bandwidth authorities, which generate a file storing information
+  on relays' measured bandwidth capacities.
+
```

Comment:
Can we have a "Format Versions" section here, like:
https://gitweb.torproject.org/torspec.git/tree/dir-list-spec.txt#n64